index.html

```diff
@@ -5,43 +5,179 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
-  <title>Document</title>
+  <title>Women in Tech</title>
 </head>
 <body>
 
-  <!-- Header Starts Here -->
-    <header>
-      <!-- Top Section -->
-      <div class="top-section">
-        <ul class="top-section-ul">
-          <li class="top-section-list">facebook</li>
-          <li class="top-section-list">facebook</li>
-          <li class="top-section-list">facebook</li>
-          <li class="top-section-list">facebook</li>
+ <main>
+   <!-- Header Starts Here -->
+   <header>
+    <!-- Top Section -->
+    <div class="top-section">
+      <ul class="top-section-ul">
+        <li class="top-section-list"><img src="Assets\icons8-facebook-18.png" alt="facebook"/></li>
+        <li class="top-section-list"><img src="Assets\icons8-twitter-18.png" alt="twitter"/></li>
+        <li class="top-section-list">English</li>
+        <li class="top-section-list">My Page</li>
+        <li class="top-section-list">Logout</li>
+      </ul>
+    </div>
+
+    <!-- Navbar -->
+    <!--Mobile-->
+    <nav class="navbar-mobile">
+      <div class="menu">
+        <a> <img src="Assets\mobile_menu 1.png" class="menubar" alt="navbar"> </a>
+         </div>
+    </nav>
+
+    <nav class="mobile-menu">
+      <ul class="mobile-menu-list">
+       <img src="Assets\icons8-close-24 (6).png" class="closebtn" alt="close">
+       <li class="menu-list"> <a href="about.html" class="menu-link">About</a></li>
+     <li class="menu-list"> <a href="#about" class="menu-link">Program</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">Join</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">Sponsor</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">News</a></li>
+    </ul>
+     </nav>
+    
+      <!-- Nav Items -->
+      <nav class="navbar-container">
+        <a href="index.html" class="p-logo"><p class="logo">WIT</p></a>
+        <ul class="navbar-items">
+          <li>
+            <a href="about.html" class="menu-link-desktop">About</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Program</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Join</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Sponsor</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">News</a>
+          </li>
+
+          <li class="list-items">
+            <a href="#">WITMP 2023</a>
+          </li>
         </ul>
-      </div>
+      </nav>
+  </header>
+<!-- Header Ends Here -->
+
+<!-- Main Starts Here -->
+<section class="theme-info">
+  <!-- Hero -->
+  <div class="first-section">
+  <div>
+  <h4 class="theme-intro">"Great News!"</h4>
+  <h2 class="theme">WOMEN IN TECH <br> MENTORSHIP PROGRAMME <br> 2023.</h2>
```

Comment:
As the previous reviewer suggested, please use a `<h1>` tag here as this is your main heading. Also please apply styles to make this heading big and bold as seen in the behance design

Your design:
<img width="949" alt="image" src="https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/37116322/ad202405-1e4d-4d32-87f3-3b1c81951eb2">

Behance design:
<img width="629" alt="image" src="https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/37116322/aa19fb4b-d3ea-404c-b754-6bd96d7c044f">

Reply:
Thank you so much for your review, I have made use of the <h1> tag as required.